src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -33,6 +33,12 @@ namespace Microsoft.Data.SqlClient
     // TODO: Add designer attribute when Microsoft.VSDesigner.Data.VS.SqlCommandDesigner uses Microsoft.Data.SqlClient
     public sealed partial class SqlCommand : DbCommand, ICloneable
     {
+        /// <summary>
+        /// Gets or sets an arbitrary tag that can be used to e.g. identify this SqlCommand in telemetry.
+        /// </summary>
+        /// <returns>The value of the tag.</returns>
+        public object Tag { get; set; }
+
```

Comment:
On the placement of this: it would ideally be placed in the higher-level `DbCommand` class, but putting it there feels like it's going to make it a bigger effort (and perhaps take much longer to be included in a deployed release). Are we fine with placing this here, in the `SqlCommand` class itself?